### ISSUE TITLE
Pretrained nets

### ISSUE THREAD

**Issue by tbuechler** [MichaelFan01/STDC-Seg#12]
Hello,

thanks for your really good work and for providing the source code! This makes some things more clear after reimplementing it.  :)

One question regarding your results on cityscape: 

I'm a bit confused about your provided pretrained weights (thanks for that, too! :)) and one sentence in your paper, but can you confirm that you have reached the mIoU of 76.8% on the Cityscape test set with STDC2-Seg75 which was only trained on the Cityscape train+val set? So no other dataset has been used for pretraining, like ImageNet for instance?

Thanks and regards,

Thomas

**Comment by MichaelFan01** [comment]
We never say we only trained on the Cityscape train+val set, I believe in our paper, we said "Following the previous methods, we use the training set and validation set to train our models before submitting to Cityscapes online server."

And also, in section 3.2.1, our first sentence is "We use the pretrained STDC networks as the backbone of our encoder and adopt the context path of BiSeNet to encode the context information." It is a common methods to use pretraind backbones in semantic segmentation tasks~
